src/integration-tests/introspector/createStaticCluster.py

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) 2018, 2020, Oracle Corporation and/or its affiliates.
```

Comment:
Does the static cluster in this test need to be dynamically added?  it could be added via off-line WLST in the existing `wl-create-domain-pod.pyt` file.  If that's fine, then there'd be no need for introspectTest.sh to be updated to rerun the introspector, and no need to run this file -- the first run of the introspector would cover the issue under test.

Reply:
The interesting thing is that creating a static cluster via off-line produces the following cluster configuration:
<cluster>
  <name>c1</name>
  <cluster-messaging-mode>unicast</cluster-messaging-mode>
</cluster>

whereas on-line WLST produces:

<cluster>
    <name>c1</name>
    <cluster-messaging-mode>unicast</cluster-messaging-mode>
    <dynamic-servers>
      <maximum-dynamic-server-count>0</maximum-dynamic-server-count>
    </dynamic-servers>
  </cluster>

The call to ClusterMBean.getDynamicServers() returns a DynamicServersMBean.  Hence, the reason I created the cluster using on-line WLST and rerun the introspector in order to test this issue.

Reply:
Oh. That's ugly and helps explain why we haven't seen the problem before. Sigh. 

To prevent someone from mistakenly 'cleaning up' the code in the future, please add a comment in this script and also in one or two key places in the main script.  The comment should be something like:   

// NOTE: The static cluster must be configured using on-line WLST instead of off-line WLST in order to reproduce OWLS 85530.

Reply:
Looks good! Thanks.